llm_bench/python/README.md

```diff
@@ -20,48 +20,44 @@ pip install -r requirements.txt
 
 ### 2. Convert a model to OpenVINO IR
    
-The conversion script for preparing benchmarking models,
-`convert.py` allows to reproduce IRs stored on shared drive.
+The optimum-cli tool allows you to convert models from Hugging Face to the OpenVINO IR format
 
 Prerequisites:
 install conversion dependencies using `requirements.txt`
 
```

Comment:
please add info that more detailed info about tool usage can be found in Optimum Intel documentation

https://huggingface.co/docs/optimum/main/en/intel/openvino/export